src/peergos/gwt/emu/java/util/function/Consumer.java

```diff
@@ -0,0 +1,16 @@
+package java.util.function;
+import jsinterop.annotations.JsType;
+
+import java.util.Objects;
+
+@FunctionalInterface
+@JsType
+public interface Consumer<T> {
+
+    void accept(T t);
+
+    default java.util.function.Consumer<T> andThen(java.util.function.Consumer<? super T> after) {
```

Comment:
I assume this is how the other pr works without _0?

Reply:
yes, we now emulate Consumer